_data/speakers.yml

```diff
@@ -659,6 +659,16 @@
   image_src: /assets/img/speakers/vickie-karasic.jpg
   image_alt: "A slightly tanned and freckled white woman with brown hair wearing rounded brown eyeglasses, looking happy and professional, and wearing a dark green top with a light gray blazer. There is a dark gray backdrop behind her."
   keynote: FALSE
+  - id: esther-jackson
```

Comment:
Please remove the space before the "-" — the data file can't be parsed if there is whitespace before the dash.